python/ray/air/checkpoint.py

```diff
@@ -318,6 +327,11 @@ def to_object_ref(self) -> ray.ObjectRef:
         Returns:
             ray.ObjectRef: ObjectRef pointing to checkpoint data.
         """
+        warnings.warn(
+            "`to_object_ref` is deprecated and will be removed. Call `ray.put` ",
+            "instead.",
```

Comment:
```suggestion
            "`to_object_ref` is deprecated and will be removed in a future Ray version. To store the checkpoint in the Ray object store, call `ray.put(ckpt)` ",
            "instead of `ckpt.to_object_ref()`.",
```